charts/payment-integration/templates/service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "payment.fullname" . }}
+  labels:
+    app: {{ template "payment.name" . }}
+    chart: {{ template "payment.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  selector:
+    app: {{ .Chart.Name }}
```

Comment:
should be `{{ template "payment.name" . }}`

Reply:
b20d9d1